LICENSE

```diff
@@ -1,3 +1,4 @@
+BSD 3-clause License
```

Comment:
BSD 3-clause licenses are not usually titles as such. See https://opensource.org/licenses/BSD-3-Clause